### ISSUE TITLE
Transform Bamboo Process model to process group

### ISSUE THREAD

**Issue by harmeet-status** [sartography/spiff-arena#881]

**Comment by burnettk** [comment]
are the bpmn files used as call activities? i guess that would seem like a reasonable reason to have multiple bpmn files in the same process model. we should also be careful (test before and after) about moving stuff that is the target of a call activity to make sure callers don't break.

**Comment by harmeet-status** [comment]
Yes they are call activities. This is a shortcut to having to create a new process model BPMN file and then copy things over which is a bit annoying.

The way I see this working is that after selecting a bunch of BPMN elements in the BPMN editor, there is a button which lets you create a new process model using the selected items. At this point, it does not delete the existing elements you have selected, you would come back and delete these once you are happy with the new process model.

**Comment by danfunk** [comment]
https://github.com/nikku/bpmn-js-copy-paste-example - is an eample of how to implement copy and paste in BPMN IO - this would allow you to select items or groups of items and copy them from one window to another window - one instance of BPMN.io to another instance - which would make authoring a BPMN process much easier.
